test/relayserver/RelayServer.test.ts

```diff
@@ -630,4 +640,85 @@ contract('RelayServer', function (accounts) {
       }
     })
   })
+
+  describe('acceptTokens', function () {
+    let testToken1 = String("0xAbCeBBc80e1a11bD4e2F692A75dFF73753aABF5f")
+    let testToken2 = String("0x85d55E6228C9a6bA73567926f0A0EB3e5f191803")
+
+    afterEach(async function(){
+      // reset verifiers for each test
+      await env.relayServer._initTrustedVerifiers([]) 
+    })
+    
+    it('should return empty if there are no trusted verifiers', async function () {
+      const res = await env.relayServer.tokenHandler()
+      assert.isEmpty(res)
+    })
+
+    it('should return error if verifier is not trusted', async function () {
+      // trust relay verifier, but query deploy verifier
+      await env.relayServer._initTrustedVerifiers([env.relayVerifier.address])
+      try{
+        const res = await env.relayServer.tokenHandler(env.deployVerifier.address)
+      }catch(error){
+        assert.equal(error.message,"supplied verifier is not trusted")
+      } 
+    })
```

Comment:
I think this test will succeed if no exception is thrown, what is actually wrong.

I think we could use [assert.throws](https://www.chaijs.com/api/assert/#method_throws)

Reply:
thanks for this, good catch. i wasn't able to use `assert.throws`, but i added an `assert.fail` statement after the token handler call, to make sure that the test doesn't pass if it doesn't throw the exception